app/design/adminhtml/Magento/backend/Magento_ProductVideo/web/css/source/_module.less

```
@@ -153,7 +153,11 @@
             &.field-video_swatch_image,
             &.field-new_video_disabled {
                 .admin__field-control {
+                    /**
+                    *@codingStandardsIgnoreStart
+                    */
```

Will it work if we'll use this kind of comment?
```suggestion
                    // @codingStandardsIgnoreStart
```

If no - pls fix the formatting to be like this

```suggestion
                    /**
                     * @codingStandardsIgnoreStart
                     */
```